src/KafkaFlow/Producers/MessageProducer.cs

```diff
@@ -360,7 +360,7 @@ void Handler(DeliveryReport<byte[], byte[]> report)
                 headers,
                 messageScopedResolver,
                 null,
-                new ProducerContext(topic, this.producerDependencyScope.Resolver));
+                new ProducerContext(topic, this.producerDependencyScope.Resolver, this.configuration.Cluster.Brokers));
```

Comment:
Should we pass the producer configurations in case in the future we need to access other configurations from the producer context?

`new ProducerContext(topic, this.producerDependencyScope.Resolver, this.configuration));`

Reply:
I kinda agree but given that `ProducerContext` and `MessageProducer` are both internal classes, I think that maybe we should only pass the configuration if we have a second use case for it. What do you say?

Reply:
Yes, I don't have an issue with that